docker/production/nginx/nginx.conf

```diff
@@ -21,6 +24,11 @@ http {
     location / {
       root /app/build;
     }
+
+    # Redirect /site/* to root
+    location ^~ /site {
```

Comment:
Outdated, need to update production to route site at root. Do we need to list the paths here?